src/extensions/maui/Microsoft.DotNet.UpgradeAssistant.Extensions.Maui/MauiPlatformTargetFrameworkUpgradeStep.cs

```diff
@@ -54,9 +54,10 @@ protected override async Task<UpgradeStepApplyResult> ApplyImplAsync(IUpgradeCon
                 if (string.Equals(projectproperties.GetProjectPropertyValue("TargetFramework").FirstOrDefault(), TargetFrameworkMoniker.NetStandard20.ToString(), StringComparison.OrdinalIgnoreCase))
                 {
                     projectproperties.RemoveProjectProperty("TargetFramework");
+                    file.SetPropertyValue("UseMaui", "true");
                     file.SetPropertyValue("TargetFrameworks", "net6.0-android;net6.0-ios");
-                    Logger.LogInformation("Added TFMs to .NET MAUI project");
                     await file.SaveAsync(token).ConfigureAwait(false);
+                    Logger.LogInformation("Added TFMs to .NET MAUI project");
                     return new UpgradeStepApplyResult(UpgradeStepStatus.Complete, $"Added TFMs to .NET MAUI Head project ");
```

Comment:
Looks like you maybe intended to include the project name here but didn't? Otherwise
```suggestion
                    return new UpgradeStepApplyResult(UpgradeStepStatus.Complete, "Added TFMs to .NET MAUI Head project");
```